network-api/networkapi/wagtailpages/migrations/0089_auto_20191105_2216.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.5 on 2019-11-05 22:16
```

Comment:
Since this PR introduces both 0088 and 0089, we should be able to make that a single migration